packages/launchpad/src/setup-actions.ts

```diff
@@ -123,6 +124,16 @@ export const setUpActionsForTasks = ( {
 						window.location.assign( `/subscribers/${ siteSlug }` );
 					};
 					break;
+				case 'site_launched':
```

Comment:
I think we also need a check in the other branch of the `if ( uiContext === 'calypso' && task.calypso_path !== undefined ) {` check above. Or we can never site the path callback for this task. 😁 But if we do that we should add some comments on the back end.